src/dev/flang/fe/LibraryOut.java

```diff
@@ -840,6 +844,7 @@ else if (e instanceof AbstractMatch m)
    */
         var cs = m.cases();
```

Comment:
Why not `m.type().isVoid()`?

Reply:
try this please, would be good to get rid of `Stop` by just checking the result type.